docs/DeveloperGuide.md

```diff
@@ -1408,16 +1408,28 @@ A lot of effort was required to make this feature work as there was very a steep
 Integrating the interface into our app with JavaFX was also extremely difficult, as the learning curve was just as steep with JavaFX and FXML elements.
 It took two weeks (from v1.1 to v1.2b) of learning and trial-and-error to achieve a working prototype of a weekly calendar and the CenterPanel which houses it.
 Furthermore, additional complex calendar logic had to be added to customise behaviours to fit our target audience, such as allowing recurrence exceptions.
-Implementation of these features took ~4k Loc.
+Implementation of these features took ~4k LOC.
 
 **Undo/Redo feature**
 
 **Finding students**
 
 **Lessons**
```

Comment:
use AB3 as a reference point e.g., you can explain that while AB3 deals with only one entity type, your project was harder because it deals with multiple entity types.